web-api/src/business/useCases/user/queueEmailUpdateAssociatedCasesWorker.test.ts

```diff
@@ -0,0 +1,186 @@
+import {
+  MAX_ITERATIONS,
+  queueEmailUpdateAssociatedCasesWorker,
+} from '@web-api/business/useCases/user/queueEmailUpdateAssociatedCasesWorker';
+import { RawUser } from '@shared/business/entities/User';
+import { applicationContext } from '@shared/business/test/createTestApplicationContext';
+import { mockPetitionerUser } from '@shared/test/mockAuthUsers';
+import { petitionerUser } from '@shared/test/mockUsers';
+
+function sleep(timeInMilliseconds: number) {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve(null);
+    }, timeInMilliseconds);
+  });
+}
```

Comment:
It looks like we have a function already: https://github.com/ustaxcourt/ef-cms/blob/af8e9ca30d8392ccd630cd25b57139120509f3ba/shared/src/tools/helpers.ts#L61